src/components/appNavigation/TabsView.js

```diff
@@ -53,13 +48,43 @@ type TabViewProps = {
 //   </View>
 // )
 
-const TabsView = React.memo((props: TabViewProps) => {
-  const { navigation, styles } = props
+const etoroButtons = (goToRewards, styles) => (
+  <>
+    <TouchableOpacity testID="rewards_tab" onPress={goToRewards} style={styles.marginLeft10}>
+      <Icon name="rewards" size={36} color="white" />
+    </TouchableOpacity>
+    <Appbar.Content />
+    <TouchableOpacity onPress={goToRewards}>
+      <Icon name="invite2" size={36} color="white" testID="invite_tab" />
+    </TouchableOpacity>
+    <Appbar.Content />
+  </>
+)
+
+const marketButtons = (goToMarketplace, style) => (
+  <>
+    <TouchableOpacity key="header-button-4" testID="goodmarket_tab" onPress={goToMarketplace} style={style}>
+      <Icon name="goodmarket" size={36} color="white" />
+    </TouchableOpacity>
+    <Appbar.Content />
+  </>
+)
+
+const supportButton = (goToSupport, space, style) => (
+  <>
+    <TouchableOpacity onPress={goToSupport} style={style}>
+      <Icon name="support2" size={36} color="white" testID="support_tab" />
+    </TouchableOpacity>
+    {space && <Appbar.Content />}
```

Comment:
as i can see space = true in both cases so you can remove this prop

Reply:
done